react/features/base/dialog/actionTypes.js

```diff
@@ -0,0 +1,22 @@
+import { Symbol } from '../react';
+
+/**
+ * The type of Redux action which begins a request to open a dialog.
+ *
+ * {
+ *     type: BEGIN_DIALOG_REQUEST,
+ *     component: React.Component,
+ *     props: React.PropTypes.object
+ * }
+ *
+ */
+export const BEGIN_DIALOG_REQUEST = Symbol('BEGIN_DIALOG_REQUEST');
```

Comment:
@saghul I don't think I understand your argument. We are talking about renaming only? For me it make sense to have default("abstract") actions for dialog open and close because there isn't anything else you can do with the dialog on this generic level.